transcription/transcription.go

```diff
@@ -29,3 +32,25 @@ func msgHeaders(from string, to []string, subject string) string {
 	msgHeaders := []string{fromHeader, toHeader, subjectHeader}
 	return strings.Join(msgHeaders, "\r\n")
 }
+
+// Transcribe a given file using a Sphinx jar. The fileName should be in
+// "name.wav" format and the jarName should be in "name.jar" format.
+func startTranscription(fileName string, jarName string) error {
```

Comment:
@roornelas so we are actually going to run with `./gradlew run` right? I'm down to include the "run command" in the config and then call this function with the command. @anoshaminai can you rename jarName to command?